docs/v2_migration.md

```diff
@@ -54,6 +54,14 @@ When different build tools are use, at least one layout needs to be set.
 The `src_folder` must be the same when using different layouts and should
 not depend on settings or options.
 
+## New conf_info properties
+
+As described in the documentation `self.user_info` has been depreated and you are now required to use
+`self.conf_info` to define individual properties to expose to downstream recipes.
+The [2.0 migrations docs](https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#removed-self-user-info)
+should over the technical details however for ConanCenterIndex we need to make sure there are no collisions
+`conf_info` must be named `user.<recipe_name>:<variable>`.
+
```

Comment:
an example would be nice to have here,
e.g. we have:
https://github.com/conan-io/conan-center-index/blob/ce23ad76af408b839ed5f3dc274481bb9f0b9ff7/recipes/gnu-config/all/conanfile.py#L53-L54 + https://github.com/conan-io/conan-center-index/blob/83a3d332a2d210b68581484b15cb27cec96d7620/recipes/libdaemon/all/conanfile.py#L76-L79
what's the direct replacement?

Reply:
Yes, I've been seeing a lot of V2 recipe updates which use `gnu-config`'s `user_info` items. It would be great to provide an example of how to do it and upgrade the `gnu-config` recipe at the same time.

Reply:
Are the Conan doc examples not clear?

Reply:
unfortunately, they are not very clear...
specifically, that part:
```
    def generate(self):
       my_value = self.dependencies[pkg].conf_info.get("user.myconf:foo")
```
1. what is `pkg` here? is it just "pkg" or "pkg/1.0" or something else?
2. is it only possible to access in `generate` method? we at least need in `build`
3. `user.myconf:foo` is too vague/abstract. can I just write `self.conf_info.define("CONFIG_SUB", "config.sub")`, or does it need to be `self.conf_info.define("user.CONFIG_SUB", "config.sub")`, or `self.conf_info.define("user:CONFIG_SUB", "config.sub")` or `self.conf_info.define("user.CONFIG_SUB:CONFIG_SUB", "config.sub")`? it's not fully clear which parts of the syntax `user.myconf:foo` are mandatory or optional...

Reply:
Conan docs is good, but too generic. I see no problem adding a more specific case in CCI docs.

Reply:
There's a boat load of discussion around should we even add this https://github.com/conan-io/conan-center-index/pull/12896

And I asked Space to stick it in a PR to make sure it works https://github.com/conan-io/conan-center-index/pull/13581/files#r998575640.

Reply:
I added a smaller example + captured some of the outside conversations. let me know if thats good!